src/files/delete-prompt/DeletePrompt.stories.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { action } from '@storybook/addon-actions'
+import { IntlProvider } from 'react-intl'
+import DeletePrompt from './DeletePrompt'
+
+storiesOf('Files', module)
+  .add('Delete Prompt', () => (
+    <IntlProvider locale='en'>
+      <div>
+        <DeletePrompt className='ma3' cancel={action('Cancel')} action={action('Delete')} files={0} folders={1} />
```

Comment:
@hacdias pls update the `cancel` and `action` props in the story

Reply:
Solved